admin/metabox/class-metabox-section-react.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generates and displays the React root element for a metabox section.
+ */
+class WPSEO_Metabox_Section_React implements WPSEO_Metabox_Section {
+
+	/**
+	 * Name of the section.
+	 *
+	 * @var string
+	 */
+	public $name;
+
+	/**
+	 * Content to use above the React root node.
+	 *
+	 * @var string
+	 */
+	public $content;
+
+	/**
+	 * Content to use for the link.
```

Comment:
reading this for the first time, as a developer who doesn't know anything about how the metabox works, I wouldn't know what this "link" is.

Reply:
As this is implementing the `WPSEO_Metabox_Tab` interface, which has two methods "link" and "content", I want to keep the naming consistent with that requirement.

I agree that that initially was not the best naming choice, but not something we should change here and now.